var/spack/repos/builtin/packages/spglib/package.py

```diff
@@ -52,6 +52,9 @@ class Spglib(CMakePackage):
 
     variant("openmp", default=True, description="Build with OpenMP support", when="@1.16.2:")
     variant("fortran", default=True, description="Build Fortran interface", when="@1.16.4:")
+    variant("tests", default=False, description="Build with tests", when="@2:")
```

Comment:
```suggestion
    variant("tests", default=False, description="Build with tests", when="@2.1.0:")
```
If I'm not mistaken?

Reply:
Oh you are right. I saw a `test` folder appearing from version 2.0, but the `SPGLIB_WITH_TESTS` CMake option is only implemented from v 2.1.